.github/workflows/benchmark.yml

```diff
@@ -45,12 +44,14 @@
           - name: Build RPC
             run: |
               cargo build --release
-          - name: Build Madara
+          - name: Download Madara
             if: steps.cache.outputs.cache-hit  != 'true'
             run: |
               cd ./lib/madara
-              cargo build --release
-          
+              mkdir -p target/release
+              cd target/release
+              wget https://github.com/keep-starknet-strange/madara/releases/download/v0.1.0.experimental.3/x86_64-unknown-linux-gnu-madara
```

Comment:
are we sure they won't delete it or that this url is fixed? as for dojo

Reply:
I had talked with Oak on this, he told me that they won't be deleting it for sometime, and I think next friday they will do a release for this, I am creating an issue for this so that we don't forget to update to release.

Reply:
I have created an issue here an assigned myself, so now when they will update the issue will help us make sure we update as well -> https://github.com/kkrt-labs/kakarot-rpc/issues/453